src/components/StudentSelector/StudentSelector.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from "react";
+import { DropdownButton, Dropdown } from "react-bootstrap";
+import { Button, Row, Col } from "react-bootstrap";
+import { withFirebase } from "utils/Firebase";
+import "./StudentSelector.scss";
+
+function StudentSelector(props) {
+  const [currentDeployment, setCurrentDeployment] = useState();
+  const [studentsChecked, setStudentsChecked] = useState([]);
+  const [chooseAll, setChooseAll] = useState(
+    Array(props.deployments.length)
+      .fill()
+      .map((_, i) => false)
+  );
+  let deployments = props.deployments;
+
+  function nameSelector(deploymentAccountId) {
+    return (
+      <div className="select-all-margins">
+        <Row>
+          <div>
+            <input
+              class="move-down"
```

Comment:
what does move-down mean as a class name

Reply:
lol thats not even supposed to be there